test/recipe/Baked.Test.Recipe.Service.Application/Program.cs

```diff
@@ -31,7 +31,7 @@
             baseClaims: ["BaseA", "BaseB"]
         ),
         core: c => c.Dotnet(baseNamespace: "Baked.Test"),
-        cors: c => c.AspNetCore(Settings.Required<string>("CorsOrigin")),
+        cors: c => c.AspNetCore(Settings.Required<string>("CorsOrigin"), "http://127.0.0.1:3000"),
```

Comment:
why is this hardcoded? who uses `CorsOrigin`, you might say `CorsOrigin.UI`, `CorsOrigin.WhateverAppIsUsingThis` etc., to have more than one cors origin from config